src/main/java/net/imglib2/RealCursor.java

```diff
@@ -64,6 +64,12 @@
  * imglib Cursors, in general.
  * </p>
  * 
+ * <p>
+ * It is not guaranteed that a RealCursor will perform bounds checking.
+ * Therefore, moving a RealCursor beyond the position of its last element will
+ * result in undefined behavior.
+ * </p>
+ *
```

Comment:
(same as above)
I would modify to "...asking the position or value of a Cursor that was not advanced to its first element or was moved beyond its last element...".